my_list.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'my_enumerable'
+
+class Mylist
+  include Enumerable
+
+  def initialize(*list)
+    @list = list
```

Comment:
- [  ] I would suggest you add them this way, it is shorter and looks better and adds them to your list which is an array
```ruby
 def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```